styles/style.css

```diff
@@ -971,3 +1018,52 @@ FAKEBOOK STYLES
     display: flex;
     gap: 1em;
 }
+
+
+.onboarding-modal {
+    border: none;
+    padding: 0;
+    background: none;
+    /* display: block; */
+}
+.onboarding-box {
+    background-color: white;
+    border-radius: 8px;
+    padding: 20px;
+    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+    text-align: left;
+    width: 600px;
+    height: 255px;
+    margin: auto;
+}
+.onboarding-box p {
+    font-size: 16px;
+    margin-bottom: 20px;
+}
+.onboarding-box button {
```

Comment:
Deze styling is niet nodig. Je kan de standaard button styling gewoon gebruiken.

Reply:
oke